charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -22,7 +22,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 39.7.0
+version: 39.7.1
```

Comment:
```suggestion
version: 39.8.0
```

Reply:
Done, could you please check again, thanks